server/models/UserAvailability.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema,
+  ObjectId = Schema.ObjectId;
+
+const userAvailability = new Schema({
+  date: {
+    type: Date,
+    required: true,
+  },
+  startTime: {
+    type: String,
+    required: true,
+  },
+  endTime: {
+    type: String,
+    required: true,
+  },
+  userId: {
+    type: ObjectId,
+  },
+  added_date: {
+    type: Date,
+    default: Date.now,
+  },
+});
```

Comment:
Not quite the approach I want to take with availability but we'll get more into that when we work on that feature.